Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM openjdk:8-jre-alpine
+
+WORKDIR /scalar
+
+# The path should be relative from build/docker. Running `gradle build`
+# (provided by com.palantir.docker plugin) will copy this Dockerfile and
```

Comment:
Does it use the palantir's plugin here?  (I don't see it though)

Reply:
Any reason for not using the palantir's plugin?

Reply:
No, Reason.
It is working fine without palantir's plugin.

Reply:
Then, let's use it. It's easier when using gradle.

Reply:
Dockerfile and guild.gradle updated with `palantir` plugin